queries.sql

```diff
@@ -8,3 +8,35 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
 SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- changing species to unspecified
+ BEGIN;
+ UPDATE animals SET species='unspecified';
+ ROLLBACK;
+
+--  Updating the animal species
+ BEGIN;
+ UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+ UPDATE animals SET species='pokemon' WHERE species IS NULL;
+ SELECT * FROM animals;
+COMMIT;
+
+-- Delete Records
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+BEGIN
+DELETE FROM animals WHERE date_of_birth>'2022-01-01';
+SAVEPOINT SP1;
+UPDATE animals SET weight_kg=weight_kg*-1;
+UPDATE animals SET weight_kg=weight_kg*-1 WHERE weight_kg<0;
```

Comment:
- [ ] According to the requirements you're supposed to rollback to the savepoint in between the update transactions